docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,165 @@
+# Upgrading to Prefect 1.0
+
+Prefect 1.0 includes some important changes that may require updates to your flow and task definitions. 
+
+Most significant: support has been removed for some features previously marked as deprecated. This means flows using deprecated features may encounter errors rather than warnings.
```

Comment:
```suggestion
Importantly, many features previously marked as deprecated have been removed. This means flows using deprecated features will encounter errors rather than warnings.
```